src/NRules.Integration/NRules.Integration.Autofac/NRules.Integration.Autofac.Tests/NRules.Integration.Autofac.Tests.csproj

```diff
@@ -10,10 +10,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
-    <PackageReference Include="Moq" Version="4.15.2" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0" />
+    <PackageReference Include="Moq" Version="4.18.2" />
+    <PackageReference Include="xunit" Version="2.4.2" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
```

Comment:
@gritcsenko I agree it makes sense upgrading xunit and moq dependencies, but can you please do it as a separate PR, and for all test projects (as opposed to just this one)?

Reply:
👍 Sure, will do